### PR TITLE
server debugging

### PR DESCRIPTION
**Pull request by cwright0428**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1,37 @@
...
+const db = require("./data/db.js");
+
+const server = express();
+
+server.use(express.json());
+
+server.post("/api/users", (req, res) => {
+  const newUser = req.body;
+  console.log("newUser", newUser);
+  db.user
```
Comment (m-j-hacker) [inline index.js:12]: So your code here will not work because you're attempting to call your `add()` function from `db.user`, but take a look at your exports in your db.js file:

`module.exports = {
  find,
  findById,
  insert,
  update,
  remove,
};`

For `db.user` to be a valid way to access your helper functions, your db.js file would contain this instead:
`module.exports = {
 user = {
  find,
  findById,
  insert,
  update,
  remove,
 }
};`
In short, you can remove the `.user` from your db call

File: index.js [hunk 2]
```diff
@@ -1 +1,37 @@
...
+      res.status(201).json({ success: true, user });
+    })
+
+    .catch(({ code, message }) => {
+      res.status(code).json({ success: false, message });
+    });
+});
+
+server.get("/api/users", (req, res) => {
+  db.users
```
Comment (m-j-hacker) [inline index.js:25]: Same issue as above, there is no `users` object in the db exports that contains a function.

**Review by m-j-hacker — COMMENTED** [review]
# Project Manager Feedback

## Requested Improvements
* Fix the issues commented on in your code and you should be all set
